### PR TITLE
Update hawk and grunt-sass versions to prevent dependency on deprecated hoek version

### PR DESCRIPTION
**Pull request by mstone121**
#### Asana
https://app.asana.com/0/52243410440449/656084807774369

# Problem
There are vulnerable `hoek` versions with this projects dependencies. They come from the `hawk` and `grunt-sass` dependencies.

# Solution
Upgrade `hawk` and `grunt-sass`

### CODE REVIEW

**Comment by jwtlake** [comment]
Did you run `npm run build`? I don't see any dist files with this change.

When I tried to run it myself it seems the build is exiting successfully but I'm also seeing an error in the message output.

```
ERROR in coins-logon-widget.js from UglifyJs
SyntaxError: Unexpected token punc «,», expected punc «:» [coins-logon-widget.js:2314,19]

Done, without errors.
```

**Comment by mstone121** [comment]
I forgot they are part of the repo. When I run `npm run build` I get no errors.

**Comment by jwtlake** [comment]
huh cool. I'll try testing then with the same method we used in https://github.com/MRN-Code/coins-logon-widget/pull/11

**Comment by mstone121** [comment]
Hmm. My build didn't come out minified. How do you get it to uglify?

**Comment by jwtlake** [comment]
The logic is in the webpack.config.js file. It checks `COINS_ENV` which is probably why it's not minifying for you.

**Review by jwtlake — APPROVED** [review]
Seems to be working just fine. 👍 

I'm gonna pre-approve and you can go ahead and merge whenever you figure out the magnification issue.

**Comment by mstone121** [comment]
When you built it in the past what command did you run? I tried setting
`NODE_ENV` and `COINS_ENV` to production to no avail.

**Comment by mstone121** [comment]
Turns out the env has to be `development` in order to uglify. 

@jwtlake The new hawk version broke the old webpack version so I had to update webpack as well. This introduced some pretty major changes so you might want to look this over again.

File: webpack.config.js [hunk 1]
```diff
@@ -1,11 +1,7 @@
 var path = require('path');
 var webpack = require('webpack');
-var DedupePlugin = webpack.optimize.DedupePlugin;
```
Comment (mstone121) [inline webpack.config.js:3]: Deduping is handled by npm now.

File: webpack.config.js [hunk 2]
```diff
@@ -1,11 +1,7 @@
...
-var UglifyJsPlugin = webpack.optimize.UglifyJsPlugin;
 var pkg = require('./package.json');
 
-var isDev = process.env.COINS_ENV === 'development';
```
Comment (mstone121) [inline webpack.config.js:7]: If we don't build in production then we might as well always minify.

Comment (jwtlake) [reply webpack.config.js:7]: Sounds reasonable.

**Comment by mstone121** [comment]
@jwtlake Can I merge this?

**Comment by jwtlake** [comment]
Just finished re-testing actually. 👍

**Pull request merged**